Bug Fixes/Minor Enhancements/Changes

Crounic opened the pull request:
# Feature Addition 

CLOSES #288 
CLOSES #289 
CLOSES #290 
CLOSES #291 

## Summary
Added file uploader name to incident files
Made UPEI Icon route to dashboard
Removed victim information section, put divs in general description Fixed incident information twice to incident information continued

## Rationale
Client Request

## Design Documentation


## Changes

Added file uploader name to incident files
Made UPEI Icon route to dashboard
Removed victim information section, put divs in general description Fixed incident information twice to incident information continued

## Impact
NA

## Testing

NA

## Screenshots/Video

![image](https://github.com/user-attachments/assets/83d2cafe-5e2b-4124-97c7-5dc73b89718d)
## Checklist

- [x] Code follows the project's coding standards

- [ ] Unit tests covering the new feature have been added

- [x] All existing tests pass

- [ ] The documentation has been updated to reflect the new feature

## Additional Notes

Any additional information or context relevant to this PR.

Crounic committed:
Added file uploader name to incident files
Made UPEI Icon route to dashboard
Removed victim information section, put divs in general description
Fixed incident information twice to incident information continued

Crounic committed:
fix code style

welshy557 committed:
Adds tests for loading files and file user on incident show page

welshy557 commented on resources/js/Layouts/Partials/DesktopSidebar.tsx:
```diff
@@ -1,13 +1,17 @@
                     <div className="flex items-center justify-center">
-                        <ApplicationLogo className="w-[50%]" />
+                        <Link as="div" className="w-[50%]" href={route('dashboard')}>
+                            {' '}
```
Looks like you had a space between the tags and the linter formatted the file and put the spaces on their own line. Remove the spaces.

Crounic committed:
Added file uploader name to incident files
Made UPEI Icon route to dashboard
Removed victim information section, put divs in general description
Fixed incident information twice to incident information continued

Crounic committed:
Merge remote-tracking branch 'origin/bug-fixes-and-minor-changes' into bug-fixes-and-minor-changes

# Conflicts:
#	resources/js/Layouts/Partials/DesktopSidebar.tsx

Crounic committed:
Added file uploader name to incident files
Made UPEI Icon route to dashboard
Removed victim information section, put divs in general description
Fixed incident information twice to incident information continued

Crounic committed:
fix code style

Crounic committed:
Linter problem fix

Crounic committed:
Moved submitted at to reporting information

Crounic committed:
fix code style
